selfdrive/car/subaru/subarucan.py

```diff
@@ -76,6 +76,19 @@ def create_es_dashstatus(packer, dashstatus_msg):
 
   return packer.make_can_msg("ES_DashStatus", 0, values)
 
+def create_infotainmentstatus(packer, infotainmentstatus_msg, visual_alert):
+  values = copy.copy(infotainmentstatus_msg)
```

Comment:
already copied in carstate.py

Reply:
done. also removed the copy from dashstatus